CHANGES.md

```diff
@@ -42,6 +42,7 @@ To be released.
 
 ### Behavioral changes
 
+ -  `Swarm<T>` now ignores `BlockHash`es which already exists.  [[#461], [#484]]
```

Comment:
Isn't the message name `BlockHashes`, not `BlockHash`?

Reply:
I intended block hash instead of message, but it seems like 'block hash' is better than '`BlockHash`'.